charts/orchestrator-infra/templates/NOTES.txt

```diff
@@ -0,0 +1,38 @@
+{{/* Empty line */}}
+
+Helm Release {{ .Release.Name }} installed in namespace {{ .Release.Namespace }}.
+{{- $yes := "YES" }}
+{{- $no := "NO " }}
+{{- $serverlessOperatorInstalled := $no }}
+{{- $knativeServingInstalled := $no }}
+{{- $knativeEventingInstalled := $no }}
+{{- $serverlessLogicOperatorInstalled := $no }}
+{{- $sonataFlowPlatformInstalled := $no }}
+{{- $timeout := "--timeout=5m" }}
+
+{{- if .Values.serverlessOperator.enabled }}
+{{- $unmanagedSubscriptionExists := include "unmanaged-resource-exists" (list "operators.coreos.com/v1alpha1" "Subscription" .Values.serverlessOperator.subscription.namespace "serverless-operator" .Release.Name .Capabilities.APIVersions ) }}
+{{- if eq $unmanagedSubscriptionExists "false" }}
+{{- $serverlessOperatorInstalled = $yes }}
+{{- end }}
+{{- end }}
+
+{{- if .Values.serverlessLogicOperator.enabled }}
+{{- $serverlessLogicOperatorInstalled = $yes }}
+{{- end }}
+
+
+Components                         Installed   Namespace
+====================================================================
+Red Hat Serverless Operator         {{ $serverlessOperatorInstalled }}      {{ .Values.serverlessOperator.subscription.namespace }}
+Red Hat Serverless Logic Operator   {{ $serverlessLogicOperatorInstalled }}     {{ .Values.serverlessLogicOperator.subscription.namespace }}
+====================================================================
+{{/* Empty line */}}
+
+To manually approve the InstalPlans:
```

Comment:
Can this and the commands below be shown only if `.Values.serverless{,Logic}Operator.subscription.installPlanApproval` are `Manual`?